Client/Ecosia/UI/MyImpact/MultiplyImpactCell.swift

```diff
@@ -58,13 +61,17 @@ final class MultiplyImpactCell: UICollectionViewCell, Themeable {
         outline.trailingAnchor.constraint(equalTo: contentView.trailingAnchor).isActive = true
         outline.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 4).isActive = true
         outline.bottomAnchor.constraint(equalTo: subtitle.bottomAnchor, constant: 12).isActive = true
+        outline.bottomAnchor.constraint(equalTo: contentView.bottomAnchor).isActive = true
 
         title.leftAnchor.constraint(equalTo: outline.leftAnchor, constant: 16).isActive = true
         title.topAnchor.constraint(equalTo: outline.topAnchor, constant: 12).isActive = true
+        title.rightAnchor.constraint(lessThanOrEqualTo: icon.leftAnchor, constant: -16).isActive = true
         title.widthAnchor.constraint(lessThanOrEqualToConstant: 220).isActive = true
         
         subtitle.leftAnchor.constraint(equalTo: outline.leftAnchor, constant: 16).isActive = true
         subtitle.topAnchor.constraint(equalTo: title.bottomAnchor, constant: 8).isActive = true
+        subtitle.rightAnchor.constraint(lessThanOrEqualTo: icon.leftAnchor, constant: -16).isActive = true
+        subtitle.widthAnchor.constraint(lessThanOrEqualToConstant: 220).isActive = true
```

Comment:
just wondering why you need the width constraint as the width is already constraint by left and right anchor, isn't it?

Reply:
Good question, this is just in case the `icon` itself is closer to the `title` and `subtitle`, on a small device for example.

- The first constrain is preventing it to grow too large for `Landscape` or `iPad`.
- The second constrain is preventing it to overlap the icon for small devices on `Portrait`.

Reply:
ok 👍